### DISCUSSION TITLE
Camera support mobile

### DISCUSSION THREAD

**Discussion by christopher3242** [rafaelvcaetano/melonDS-android#1175] [Ideas]
I think question? 
![Screenshot_2023-07-01-10-51-13-025_me magnum melonds](https://github.com/rafaelvcaetano/melonDS-android/assets/52284167/1d562e7f-8ab6-496b-a8a8-88ecf7814d66)
![Screenshot_2023-07-01-10-57-49-126_me magnum melonds](https://github.com/rafaelvcaetano/melonDS-android/assets/52284167/f9346b36-7dba-4619-bf3f-409cda03cfc7)

**Comment by rafaelvcaetano** [comment]
DSi camera support is in development. You can read more about it here: https://github.com/rafaelvcaetano/melonDS-android/discussions/1093
